### PR TITLE
Feature backend/453 load base data

### LINKED ISSUE #453: Load base data on backend startup
A basic skilltype and communication template should be loaded when the backend is started.

### PR DESCRIPTION
**Pull request by KasperDemeyere**
This pr closes #453.

### CODE REVIEW

**Comment by KasperDemeyere** [comment]
> Seems good to me! Good work again Kasper! Could you just provide a little doc in this PR why you chose to change the Id generation type.

By setting the id generation type to identity, we let postgres generate the id's instead of spring. This makes it possible for us to insert data using the data.sql file without manually needing to specify id's.

File: backend/src/main/resources/data.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+INSERT INTO skill_type (colour, name) VALUES ('#632CB1', 'Video editor');
+INSERT INTO skill_type (colour, name) VALUES ('#A73C97', 'Photographer');
+INSERT INTO skill_type (colour, name) VALUES ('#505369', 'Other');
+
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('Yes', 'Accepted for OSOC',
+                    'We are pleased to tell you that you have been accepted for OSOC!\n Someone will be in touch with you shortly to work out the details.');
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('No', 'Rejected for OSOC',
+                    'Unfortunately we are unable to find a fitting spot for you in this edition of OSOC.\n You can always try again next year!');
```
Comment (andpuydt) [inline backend/src/main/resources/data.sql:19]:
```suggestion
                    'Unfortunately we were unable to find a fitting position for you in this edition of OSOC.\n You can always try again next year!');
```

File: backend/src/main/resources/data.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+INSERT INTO skill_type (colour, name) VALUES ('#1CEC1C', 'Copywriter');
+INSERT INTO skill_type (colour, name) VALUES ('#632CB1', 'Video editor');
+INSERT INTO skill_type (colour, name) VALUES ('#A73C97', 'Photographer');
+INSERT INTO skill_type (colour, name) VALUES ('#505369', 'Other');
+
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('Yes', 'Accepted for OSOC',
+                    'We are pleased to tell you that you have been accepted for OSOC!\n Someone will be in touch with you shortly to work out the details.');
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('No', 'Rejected for OSOC',
```
Comment (andpuydt) [inline backend/src/main/resources/data.sql:18]: Rejected for OSOC sounds a bit harsh as mail subject... Maybe we should change all subjects to OSOC application status to soften the blow?

File: backend/src/main/resources/data.sql [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+                    'Unfortunately we are unable to find a fitting spot for you in this edition of OSOC.\n You can always try again next year!');
+INSERT INTO communication_template (name, subject, template)
+            VALUES ('Maybe', 'OSOC application status',
+                    'We might have a spot for you in the next edition of OSOC, but we are still figuring some things out.\n Hold on tight!');
```
Comment (andpuydt) [inline backend/src/main/resources/data.sql:22]:
```suggestion
                    'We might have a spot for you in the upcoming edition of OSOC, but we are still figuring some things out.\n Hold on tight!');
```

**Review by andpuydt — COMMENTED** [review]
Looks good! Just some small remarks

**Pull request merged**